engines/director/lingo/lingo.cpp

```diff
@@ -1228,6 +1234,8 @@ const char *Datum::type2str(bool ilk) const {
 		return ilk ? "object" : "OBJECT";
 	case PARRAY:
 		return ilk ? "proplist" : "PARRAY";
+	case PICTUREREF:
+		return "PICTUREREF";
```

Comment:
I checked with the original, this code should be as follows
```suggestion
		return ilk ? "picture" :  "PICTUREREF";
```